src/packages/core/tree/tree.element.ts

```diff
@@ -17,29 +23,21 @@ export class UmbTreeElement extends UmbLitElement {
 		this.#treeContext.setTreeAlias(newVal);
 	}
 
-	@property({ type: Boolean, reflect: true })
-	get selectable() {
-		return this.#treeContext.selection.getSelectable();
-	}
-	set selectable(newVal) {
-		this.#treeContext.selection.setSelectable(newVal);
-	}
-
-	@property({ type: Array })
-	get selection() {
-		return this.#treeContext.selection.getSelection();
-	}
-	set selection(newVal) {
-		if (!Array.isArray(newVal)) return;
-		this.#treeContext?.selection.setSelection(newVal);
-	}
+	private _selectionConfiguration: UmbTreeSelectionConfiguration = {
+		multiple: false,
+		selectable: true,
+		selection: [],
+	};
 
-	@property({ type: Boolean, reflect: true })
-	get multiple() {
-		return this.#treeContext.selection.getMultiple();
+	@property({ type: Object })
```

Comment:
We have not been too adamant about this, but to eventually support Lit 3, we need to get used to putting the `@property` declarations on the **setter** rather than the getter.

Reply:
@iOvergaard Thanks, good to know.
Should I move the `@property` down to the setter, or move the setter to be before the getter?
_(My C# mindset thinks getter first.)_

Reply:
> Does Lit have a recommendation?

The Lit 3 upgrade guide has an example of the setter being first, but it implies that this has always been the case (for previous Lit version), so I guess it's up to our coding style preferences? _No strong preference from me._
https://lit.dev/docs/releases/upgrade/#decorated-getter

Reply:
Reordered the `@property` setter methods in commit d670111800467c8408b0d6dfc840edac9b66bace.